lib/tasks/vacancies_to_update.yaml

```diff
@@ -212,3 +213,485 @@ We can offer:
+      <p>Sacred Heart is outstanding in its inclusion policy and practice, making every effort in our mission to deliver the very best for all the students in our care.  Sacred Heart is a converted Academy which as a school was judged “Outstanding” by OfSTED in 2011 and holds a number of nationally recognised awards including: Leading Parent Partnership and Investors in Careers.  The Academy was inspected for Section 48 in May 2017 and was judged to be “Outstanding”, the inspectors noted, <em>“excellent relationships exist between staff and pupils, which is evident throughout the school community and facilitates the spiritual growth of the individual and the whole academy.”</em></p>
+      <p>The Academy is committed to working in partnership and collaboration with other schools and colleges in the sector and with our partner primaries in the Nicholas Postgate Academy Trust.  This affords staff opportunities for networking and sharing good practice with others in secondary, primary and special schools.</p>
+      <p>The successful candidate will be expected to have the ability to teach across the full 11-16 age and ability range. This is an excellent opportunity for an individual with energy and vision to join us in our drivefor excellence.</p>
+      <p>We are committed to safeguarding children and this post is subject to Enhanced Disclosure and Barring Service check.  Applications should be submitted with a covering letter stating why you would be suitable for this post and enclosing a completed Application Form, Recruitment Monitoring Form and Rehabilitation of Offenders Act – Disclosure Form to recruitment@sacredheartrc.org.uk </p>
+      <p>Closing Date: Monday 14th May 2018 at noon<br> Interviews will take place on Thursday 17th May  2018</p>"
+      education: "The successful candidate will be someone who has:
+      <ul><li>a passion for providing the best possible education for all students</li>
+      <li>a commitment to maximising achievement through excellence in teaching and learning</li>
+      <li>a strong commitment to the Catholic ethos of the Trust and the promotion of Christian values</li>
+      <li>a commitment to continual improvement</li>
+      <li>the ability to inspire students and promote a sense of enjoyment of learning</li>
+      <li>an eagerness to be involved with extra-curricular activities in order to enhance the lives of the whole community</li></ul>"
+    -
+      slug: 'teacher-of-physics'
+      job_descritpion: "
```

Comment:
This is the entry that my go of the rake task crashed on.

Reply:
🙄